README.md

```diff
@@ -14,7 +14,7 @@ terraform {
   required_providers {
     ct = {
       source  = "poseidon/ct"
-      version = "0.11.0"
```

Comment:
Let's wait until closer to release before starting to change the version

Reply:
Ok, remove the change